lib/index.js

```diff
@@ -0,0 +1 @@
;color:",";}.icon-wrapper{position:absolute;top:0;right:0;padding:0 8px;height:100%;display:flex;align-items:center;}.icon-container{position:relative;top:1px;}"],b.black,(function(e){return e.bordered?"2px solid ".concat(b.black):"none"}),(function(e){return e.selected&&"\n    background-color: ".concat(b.black,";\n    color: ").concat(b.white,";\n")}),b.black,b.white);function _e(e,n){return function(e){if(Array.isArray(e))return e}(e)||function(e,n){if(!(Symbol.iterator in Object(e)||"[object Arguments]"===Object.prototype.toString.call(e)))return;var t=[],r=!0,o=!1,a=void 0;try{for(var i,l=e[Symbol.iterator]();!(r=(i=l.next()).done)&&(t.push(i.value),!n||t.length!==n);r=!0);}catch(e){o=!0,a=e}finally{try{r||null==l.return||l.return()}finally{if(o)throw a}}return t}(e,n)||function(){throw new TypeError("Invalid attempt to destructure non-iterable instance")}()}var Ge=function(e){var n=_e(Object(a.useState)(e.isOpened||!1),2),t=n[0],r=n[1],o=_e(Object(a.useState)(""),2),l=o[0],c=o[1],s=Object(a.useRef)(),u=Object(a.useRef)(),d=e.showResetStyledControls&&e.selectedValue,g=s.current?s.current.offsetTop:0,p=function(e){!t&&u.current.contains(e.target)?r(!0):t&&r(!1)},f=function(n){n.stopPropagation(),e.resetSelectedValue(e.filterId)};return Object(a.useEffect)((function(){if(e.selectedValue&&e.options.length){var n=e.options.find((function(n){return n&&(String(n.id)===e.selectedValue||n.name&&n.name.toLowerCase()===e.selectedValue||n&&n.ids===e.selectedValue)}));c(n?n.name:"")}else c("")}),[e.options,e.selectedValue]),Object(a.useEffect)((function(){return window.addEventListener("click",p),function(){window.removeEventListener("click",p)}})),i.a.createElement(De,null,i.a.createElement(Se,{ref:s,isFetching:e.isFetching,role:"option",tabIndex:0,id:"dropdown__".concat(e.id),"data-role":"dropdown-filter"},e.customPlaceholder?e.renderCustomPlaceholder(u,t):i.a.createElement(Te,{"data-testid":"selected-".concat(l),ref:u,isOpened:t,"data-role":"dropdown-value-display",selectType:e.selectType,"data-opened":t,isFilterSelected:l},l||e.label,i.a.createElement(F,{icon:"dropdown-arrow",width:22,height:22})),i.a.createElement(Pe,{offset:g+45,isVisible:t},d&&i.a.createElement(Re,{role:"option",tabIndex:0,key:"reset",value:"@@resetFilter",onClick:f},"View all"),e.options.length?e.options.reduce((function(n,t){if(t){var r=t.name,o=t.id,a=t.ids||String(o);n.push(i.a.createElement(Ye,{key:a,"data-testid":"option-".concat(a),role:"option",tabIndex:0,bordered:!1,selected:a===e.selectedValue,onClick:function(){return e.handleSelectChange(e.filterId,a)},"data-role":"dropdown-option"},r,e.showResetStyledControls&&a===e.selectedValue&&i.a.createElement("div",{onClick:f,"data-testid":"cancel-".concat(a)},i.a.createElement(F,{width:10,height:10,fillColor:"white",icon:"close"}))))}return n}),[]):i.a.createElement(Be,null,"None"))))};Ge.propTypes={resetSelectedValue:o.a.func.isRequired,showResetStyledControls:o.a.bool,label:o.a.string,customPlaceholder:o.a.bool,renderCustomP
```

Comment:
I don't think we want to commit the `lib` folder? We can generate it during the build?